lib/promscrape/config.go

```diff
@@ -1197,7 +1197,7 @@ func (swc *scrapeWorkConfig) getScrapeWork(target string, extraLabels, metaLabel
 	var at *auth.Token
 	tenantID := labels.Get("__tenant_id__")
 	if len(tenantID) > 0 {
-		newToken, err := auth.NewToken(tenantID)
+		newToken, err := auth.NewTokenPossibleMultitenant(tenantID)
```

Comment:
Could you explain a bit why a user would add the `__tenant_id__="multitenant"` label and what the expectation is(`multitenant` equals the default tenant)? Or it's just for compatiblity?
But vminsert doesn't recognize `multitenant` in `vm_account_id` or `vm_project_id` labels.

Reply:
literally my question posted a few seconds after yours :)

Reply:
yep, your network beat me, sorry nik for the dup:)

Reply:
`__tenant_id__="multitenant"` is equal to empty tenantID label. This change is only needed for consistency of tenantID format

Reply:
I'm fine to answer the same question multiple times:)